frontend/components/common/other/BasePanel.vue

```diff
@@ -35,6 +36,12 @@ const STATES = [
         iconClass: 'text-red-500',
         isExpanded: true,
     },
+    {
+        name: 'partlyIncorrect',
```

Comment:
Let's make it `attention` so it can be used in other places
```suggestion
        name: 'attention',
```